pt/lessons/basics/strings.md

```diff
@@ -14,37 +16,38 @@ Strings em Elixir são nada mais que uma sequência de bytes. Vamos ver um exemp
 ```elixir
 iex> string = <<104,101,108,108,111>>
 "hello"
+iex> string <> <<0>>
+<<104, 101, 108, 108, 111, 0>>
 ```
 
+Ao concatenar a string com o byte `0`, o IEx mostra a string como um binário já que este não é mais uma string válida. Este truque nos ajuda a identifcar a sequência de bytes que compõem qualquer string.
+
 >NOTA: Ao usar a sintaxe com << >> estamos dizendo ao compilador que os elementos dentro desses símbolos são bytes.
 
 ## Listas de Caracteres
 
 Internamente, as strings em Elixir são representadas como uma sequência de bytes ao invés de um array de caracteres. Elixir também tem um tipo char list (lista de caracteres). Strings em Elixir são delimitadas por aspas duplas, enquanto listas de caracteres são delimitadas por aspas simples.
 
-Qual a diferença? Cada valor de uma lista de caracteres é o valor ASCII do caractere. Vamos ver isso mais a fundo:
+Qual a diferença? Cada valor em uma lista de caracteres corresponde ao número Unicode do caracter, enquanto em um binário os valores são codificados em UTF-8. Vamos ver isso mais a fundo:
 
 ```elixir
-iex> char_list = 'hello'
-'hello'
-
-iex> [hd|tl] = char_list
-'hello'
+iex(5)> 'hełło'
+[104, 101, 322, 322, 111]
+iex(6)> "hełło" <> <<0>>
+<<104, 101, 197, 130, 197, 130, 111, 0>>
+```
 
-iex> {hd, tl}
-{104, 'ello'}
+`322` é o número Unicode de ł, representado em UTF-8 pelos dois bytes `197`, `130`.
 
-iex> Enum.reduce(char_list, "", fn char, acc -> acc <> to_string(char) <> "," end)
-"104,101,108,108,111,"
-```
+Ao programar em Elixir, geralmente usamos strings ao invés de listas de caracteres. O suporte a listas de caracteres é incluso principalmente por ser obrigatório para alguns módulos Erlang.
 
-Ao programar em Elixir, geralmente usamos Strings ao invés de listas de caracteres. O suporte a listas de caracteres é incluso principalmente por ser obrigatório para alguns módulos Erlang.
+Para mais informação, veja o oficial [`Guia de Introdução`](http://elixir-lang.org/getting-started/binaries-strings-and-char-lists.html).
```

Comment:
It looks to me that `official` here is a bit misplaced. Maybe placing it at the end of the sentence? `... veja o Guia de Introdução oficial.`